packages/cli/src/commands/bundle.js

```diff
@@ -22,11 +23,12 @@ export const bundleCommand = async ({
         format: 'endoZipBase64',
       })
     );
+  const bundlePath = bundleName && parsePetNamePath(bundleName);
```

Comment:
Conventionally, we've only treated `undefined` values as absent:
```suggestion
  const bundlePath = bundleName === undefined ? bundleName : parsePetNamePath(bundleName);
```

Reply:
That makes sense.  I was following the pattern of some of the other commands that had previously been converted to use paths, and it looked like the pattern
```
const fooPath = fooName && parsePetNamePath(fooName);
```
was an established idiom.  If we want to look for explicit `undefined` values then probably those older bits should be updated to match.

Operationally though, these get parsed from the command line by `commander`, which means that the value is always going to be either `undefined` or an actual string, so it's not obvious to me that the additional verbosity buys us much of value.  There is some suspenders-and-belt protection in the event that command implementations get used compositionally in the implementation of other commands, but that's not something I see in the current code and is probably a questionable practice.

Reply:
After taking a closer look, it turned out that there is only (now) one other occurrences of this pattern, in `run.js`.  However, I think my observation about `commander` processing the args is still apt, so I think a better pattern is to leave the current parse line unchanged but precede it with
```
assert(fooName === undefined || typeof fooName === 'string');
```
I think this will be a little easier on the eyes while still requiring that the invariant that would normally be ensured by `commander` remains true even if the command function is called directly without the command line parse via `commander` as an intermediary.